fastlane/Fastfile

```diff
@@ -251,10 +254,10 @@ private_lane :update_native_version do |options|
   if dry_run
     UI.message("ℹ️  Nothing more to do, dry_run: true")
   else
-    replace_text_in_files(
-      previous_text: previous_version_number,
-      new_text: new_version_number,
-      paths_of_files_to_update: files_to_update
+    replace_version_number(
```

Comment:
We need to fix this action to take a hash instead of arrays so it supports patterns